sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/MicroBatchExecution.scala

```diff
@@ -171,7 +173,9 @@ class MicroBatchExecution(
             // Materialize source to avoid creating it in every batch
             val metadataPath = s"$resolvedCheckpointRoot/sources/$nextSourceId"
             nextSourceId += 1
-            logInfo(s"Reading table [$table] from DataSourceV2 named '$srcName' $dsStr")
+            logInfo(log"Reading table [${MDC(STREAMING_TABLE, table)}] " +
```

Comment:
I wonder if we need the STREAMING prefix for these variables. But we can always revisit and combine variables later.

Reply:
Also for streaming we have streaming query id which is useful in identifying which log is from which query. This is particularly useful when there are multiple streaming queries. Was wondering if this is worth added.